Cleanup, homepage css grid init, logo svg, sirv, form styles, typo & typing fixing

aretecode opened the pull request:
also...
improvements to font loading
adding manifest

aretecode committed:
move import out of AppStyles (inline font load)

aretecode committed:
docs: future features, comment on auth0 work

aretecode committed:
styles: Logo (use svg, consistent white vs #fff)

aretecode committed:
create Scribe svg icon

ugly right now, will be polished

aretecode committed:
Merge branch 'feature/graphql' of github.com:aretecode/konsilos-v2-help into cleaning

aretecode committed:
meta/config: migrate to sirv, add manifest

- move images to sirv
- add manifest.json
- update urls for preconnect & prefetch fonts
- add todo for responsive images using sirv or pic

aretecode committed:
typings for serialized as any for now

aretecode committed:
styling: form as per material design (partial)

aretecode committed:
styling: tweak app level styles (page pg, vars)

aretecode committed:
! styling: css grid transform

aretecode committed:
meta/config: pkg description & urls

aretecode committed:
Merge branch 'feature/graphql' of github.com:aretecode/konsilos-v2-help into cleaning

aretecode committed:
typings for user & minor typo fixing

 (familyMembers => familyList)

aretecode commented:
# previous
<img width="1243" alt="Screen Shot 2019-09-21 at 19 49 25" src="https://user-images.githubusercontent.com/4022631/65381572-ef5d5b00-dca8-11e9-8c3c-f0d10abb7e2c.png">

# after
<img width="1243" alt="Screen Shot 2019-09-21 at 19 49 23" src="https://user-images.githubusercontent.com/4022631/65381573-f2f0e200-dca8-11e9-9b2b-ae0ff5d5e6dc.png">

understood-devops-2 committed:
fix for top level advices families

aretecode committed:
autofix format

mfrohberg reviewed (approved):
Please merge this in https://github.com/aretecode/konsilos/commit/d98d73e52e27d63961fb2d3485b9b8f29f7bfe9e

aretecode committed:
components & app: add minimal Footer

aretecode committed:
styles: unused line & use css-var on welcome page

aretecode committed:
components: LanguageSwitcher - pt 2 (minimal)

- use simpler flags
- make flags come in correct order
- make actual switcher functionality
- and & update todos

aretecode committed:
styles: LanguageSwitcher flag sizing

aretecode committed:
styling: tweaking responsive devices

aretecode committed:
styling: flag width & height

aretecode committed:
svg: html props => react props

aretecode committed:
styling: list styles & props for FamilyList Avatars

aretecode committed:
Merge pull request #9 from aretecode/feature/language-switcher

components: LanguageSwitcher - pt 2 (minimal)

aretecode committed:
Merge branch 'feature/graphql' of github.com:aretecode/konsilos-v2-help into feature/language-switcher

aretecode committed:
components: Add Nextjs StyledLink

aretecode committed:
pages: Family - add h1

aretecode committed:
components: Footer - add simple nav

aretecode committed:
style: remove old list-style from FamilyList

now it is in AppStyles

aretecode committed:
Merge pull request #10 from aretecode/merge-for-merge-style-fixes

Link & style tweak & top level advices fix merged

aretecode commented:
@mfrohberg ^ done
